prow/plugins/verify-owners/verify-owners.go

```diff
@@ -224,14 +328,106 @@ func parseOwnersFile(b []byte, c github.PullRequestChange, log *logrus.Entry, la
+	var commentLines []string
+	commentLines = append(commentLines, fmt.Sprintf(nonCollaboratorResponseFormat, ownersFileName, org))
+
+	for user, ownersFiles := range nonTrustedUsers {
+		commentLines = append(commentLines, fmt.Sprintf("- %s", user))
+		for _, filename := range ownersFiles {
+			commentLines = append(commentLines, fmt.Sprintf("  - %s", filename))
 		}
 	}
-	return nil
+	return strings.Join(commentLines, "\n")
+}
+
+func nonTrustedUsersInOwnersAliases(ghc githubClient, log *logrus.Entry, triggerConfig plugins.Trigger, org, repo, dir, patch string, ownerAliasesModified, skipTrustedUserCheck bool) (map[string][]string, repoowners.RepoAliases, error) {
+	repoAliases := make(repoowners.RepoAliases)
+	// nonTrustedUsers is a map of non-trusted users to the list of files they are being added in
+	nonTrustedUsers := map[string][]string{}
+	var err error
+
+	// If OWNERS_ALIASES exists, get all aliases.
+	path := filepath.Join(dir, ownersAliasesFileName)
+	if _, err := os.Stat(path); err == nil {
+		b, err := ioutil.ReadFile(path)
+		if err != nil {
+			return nonTrustedUsers, repoAliases, fmt.Errorf("Failed to read %s: %v", path, err)
+		}
+		repoAliases, err = repoowners.ParseAliasesConfig(b)
+		if err != nil {
+			return nonTrustedUsers, repoAliases, fmt.Errorf("error parsing aliases config for %s file: %v", ownersAliasesFileName, err)
+		}
+	}
+
+	// If OWNERS_ALIASES file was modified, check if newly added owners are trusted.
+	if ownerAliasesModified && !skipTrustedUserCheck {
+		allOwners := repoAliases.ExpandAllAliases().List()
+		for _, owner := range allOwners {
+			// cap the number of checks to avoid exhausting tokens in case of large OWNERS refactors.
+			if len(nonTrustedUsers) > 20 {
+				break
+			}
+			nonTrustedUsers, err = checkIfTrustedUser(ghc, log, triggerConfig, owner, patch, ownersAliasesFileName, org, repo, nonTrustedUsers, repoAliases)
+			if err != nil {
+				return nonTrustedUsers, repoAliases, err
+			}
+		}
+	}
+
+	return nonTrustedUsers, repoAliases, nil
+}
+
+func nonTrustedUsersInOwners(ghc githubClient, log *logrus.Entry, triggerConfig plugins.Trigger, org, repo, patch, fileName string, owners []string, nonTrustedUsers map[string][]string, repoAliases repoowners.RepoAliases) (map[string][]string, error) {
+	var err error
+	for _, owner := range owners {
+		// cap the number of checks to avoid exhausting tokens in case of large OWNERS refactors.
+		if len(nonTrustedUsers) > 20 {
+			break
+		}
+
+		// ignore if owner is an alias
+		if _, ok := repoAliases[owner]; ok {
+			continue
+		}
+
+		nonTrustedUsers, err = checkIfTrustedUser(ghc, log, triggerConfig, owner, patch, fileName, org, repo, nonTrustedUsers, repoAliases)
+		if err != nil {
+			return nonTrustedUsers, err
+		}
+	}
+	return nonTrustedUsers, nil
+}
+
+// checkIfTrustedUser looks for newly addded owners by checking if they are in the patch
+// and then checks if the owner is a trusted user.
```

Comment:
`... because ...`? As in, as an ignorant reader, I'm not sure why this is needed. I can guess, but don't know from the comment.

Reply:
We require people to be org members to be able to lgtm and approve. So we want to make sure that if anyone is added to an OWNERS file, they are an org member.